index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>code review</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="section">
+    <h1 id="title">Survey Form Designed by Khalil & Sheldon</h1>
+    <p id = "description">We are going to design this survey form for microverse admisssion trials</p>
```

Comment:
Please I would suggest you add this section to the `header` tag for semantic reasons. This will make your code clean and easy to fix bugs.